sources/api/migration/migrations/v1.11.0/aws-creds-metadata/src/main.rs

```diff
@@ -0,0 +1,23 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::common_migrations::{AddMetadataMigration, SettingMetadata};
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added a new setting for AWS credential configuration.
+fn run() -> Result<()> {
+    migrate(AddMetadataMigration(&[SettingMetadata {
+        metadata: &["affected-services"],
+        setting: "settings.aws",
```

Comment:
Here's that mysterious service again.

```
        setting: "settings.aws",
```

Reply:
I forget the details now, but I was told this was needed to get the plumbing of everything working right.